test/MyFirstApiConsume.test.js

```diff
@@ -2,52 +2,75 @@ const axios = require("axios");
 const { expect } = require("chai");
 const { StatusCodes } = require("http-status-codes");
 
-describe("First Api Tests", () => {});
+describe("First Api Tests", () => {
+    it("Consume GET Service", async () => {
+        const response = await axios.get("https://httpbin.org/ip");
 
-it("Consume GET Service", async () => {
-    const response = await axios.get("https://httpbin.org/ip");
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property("origin");
+    });
 
-    expect(response.status).to.equal(StatusCodes.OK);
-    expect(response.data).to.have.property("origin");
-});
+    it("Consume GET Service with query parameters", async () => {
+        const query = {
+            name: "John",
+            age: "31",
+            city: "New York",
+        };
 
-it("Consume GET Service with query parameters", async () => {
-    const query = {
-        name: "John",
-        age: "31",
-        city: "New York",
-    };
+        const response = await axios.get("https://httpbin.org/get", { query });
 
-    const response = await axios.get("https://httpbin.org/get", { query });
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
 
-    expect(response.status).to.equal(StatusCodes.OK);
-    expect(response.config.query).to.eql(query);
-});
+    it("Consume HEAD Service", async () => {
+        const response = await axios.head("https://httpbin.org/anything");
 
-it("Consume HEAD Service", async () => {
-    const response = await axios.head("https://httpbin.org/anything");
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.headers).to.have.property("date");
+    });
 
-    expect(response.status).to.equal(StatusCodes.OK);
-    expect(response.headers).to.have.property("date");
-});
+    it("Consume PATCH Service", async () => {
+        const myData = {
+            name: "Daniel",
+            lastname: "Calle",
+            github: true,
+        };
 
-it("Consume PATCH Service", async () => {
-    const response = await axios.patch("https://httpbin.org/anything");
+        const response = await axios.patch("https://httpbin.org/patch", {
+            myData,
+        });
```

Comment:
in this case, as the method supports the request body, you don't need to pass the data in this format `{ myData }`, instead of that, as you already defined the myData variable in JSON format, you can pass it without brackets:

`axios.patch("https://httpbin.org/patch", myData);`